ta/os_test/os_test.c

```diff
@@ -980,6 +980,8 @@ static void undef_instr(void)
 	__asm__(".word 0x0");
 #elif defined(ARM32)
 	__asm__(".word 0xe7ffffff");
+#elif defined(RV64) || defined(RV32)
+	__asm__(".word 0x0");
```

Comment:
don't want to mix arm and riscv? :)
```patch
-#if defined(ARM64)
+#if defined(ARM64)|| defined(RV64) || defined(RV32)
 	__asm__(".word 0x0");
 #elif defined(ARM32)
```

Reply:
For better code reading, I suggest to keep it as it.